src/styles/Colors.ts

```diff
@@ -69,6 +69,7 @@ enum Colors {
   amber500 = '#F59E0B',
   amber600 = '#D97706',
   amber900 = '#78350F',
+  amber90099 = '#78350F99',
```

Comment:
Is `'#78350F99` supported? Shouldn't it be RGBA?

Reply:
Ahh, actually it should be just deleted as it's not used :)

Reply:
It's used in Cloud and used directly as is
<img width="264" alt="image" src="https://github.com/kubeshop/testkube-dashboard/assets/25991946/d96296b1-6e07-421c-8357-0176ba89a8c2">